docs/contributing/pull-requests/feature-branches.md

```diff
@@ -0,0 +1,51 @@
+# Long-lived Feature Branches
+
+A long lived feature branch differ from regular Pull Requests in three significant ways.
+
+1. Has multiple authors.
+2. Consists of multiple already reviewed Pull Requests.
+3. Only consists of merge commits.
```

Comment:
```suggestion
1. It has multiple authors.
2. It consists of multiple already reviewed Pull Requests.
3. It only consists of merge commits.
```

Reply:
What do you think about changing the the leading sentance and the lists to:

```
A long lived feature branch differs from a regular Pull Requests in three significant ways. It
should:

- Have multiple authors.
- Consists of multiple already reviewed Pull Requests.
- Only consists of merge commits.
```

I also changed the list from numbered since the order doesn't matter.

Reply:
Yes, also a good solution. I'd also drop the full-stops at the end of each dot point but that's up to you.

Reply:
I've left a new review below